### DISCUSSION TITLE
[Announcement] We just launched Heroicons.com!

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2238] [Show and tell]
A few months back we quietly released [Heroicons](https://www.heroicons.com/), a set of free SVG icons we initially designed to support the components in Tailwind UI.

Today we launched the official [Heroicons web experience](https://www.heroicons.com/), which makes it easier than ever to search for icons and quickly copy them to your clipboard as Tailwind-ready HTML or JSX.

[![image](https://user-images.githubusercontent.com/4323180/91183921-0e831a80-e6ba-11ea-81d7-6c86eb9ccaa8.png)](https://heroicons.com)

There are currently over 220 icons available in both medium and small sizes, with size designed to serve a different use-case:

- Medium icons are designed to be rendered at 24x24, and work well for things like primary navigation and marketing sections.
- Small icons are designed to be rendered at 20x20, and work well for buttons, form elements and to support text.

All of the icons are Tailwind-ready, and are easy to style with Tailwind’s built-in size and color utilities.

```html
<svg
  class="h-6 w-6 text-indigo-500"
  xmlns="http://www.w3.org/2000/svg"
  fill="none"
  viewBox="0 0 24 24"
  stroke="currentColor"
>
  <path
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
    d="M17 8l4 4m0 0l-4 4m4-4H3"
  />
</svg>
```

Check them out and let us know what you think!

**Comment by royvanv** [comment]
Looking great!

Would be nice if you include a `npm install` command somewhere on the page.

**Reply by adamwathan** [reply]
Thanks! We will at some point, right now we don't consider what we have published to npm to be stable or production-ready enough to really advertise.

**Comment by jenstornell** [comment]
Great work! I really like that you also provide 20px icons. It's not very common, but very useful!

Nice website as well!

The only nag I have is that it took me a while before I found the search input field. It's not very visible. It blends in too much with the text below.

I tried to improve it in just 1 min (thanks to Tailwind classes), just added a gray background behind it. Now it's much more visible and stands out from the text. It looks a bit like crap because I just mocked it in, but I think you get the point.

What do you think?

![heroicons com_](https://user-images.githubusercontent.com/927011/91310925-c0057700-e7b2-11ea-8a05-755f41263489.png)

**Comment by jonathan-hildebrandt** [comment]
Hi, I want to create a few of these svg icons on my own, but i dont really now how to do this. Could you tell me what software you are using to create these very clean icons?
